spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+# Specs in this file have access to a helper object that includes
+# the AdvertisementsHelper. For example:
+#
+# describe AdvertisementsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+
+RSpec.describe ApplicationHelper, "#avatar_url" do
+    it "displays avatar url by passing in a user" do
+      user = build(:user)
+      url = helper.avatar_url(user)
```

Comment:
where's the helper I should compare this to? shouldn't it take in a pixels to get the right sized gravatar? what if the user doesn't have a gravatar, how do you handle it?